inspirehep/modules/workflows/errors.py

```diff
@@ -26,5 +26,9 @@
 
 
 class DownloadError(WorkflowsError):
-
     """Error representing a failed download in a workflow."""
+
+
+class MissingHeadUUIDError(WorkflowsError):
+    """Error representing the missing field `head_uuid` inside the property
+    extra_data for a given workflow object."""
```

Comment:
You are right! Anyway It would be nice add a doc string checker to our tests instead of link every time the python doc. Because I think after a while can be frustrating comment the doc strings.
Just to avoid this kind of errors that can be prevented during the development phase without the help of a human... :smile: